pages/dkp/kommander/2.2/workspaces/applications/platform-applications/platform-application-requirements/index.md

```diff
@@ -14,6 +14,8 @@ Workspace platform applications require more resources than solely deploying or
 
 The following table describes all the workspace platform applications that are available to the clusters in a workspace, minimum resource requirements, and whether they are enabled by default.
 
+<need to update this table for 2.2>
+
 | Name | Minimum Resources Suggested | Minimum Persistent Storage Required | Deployed by Default |
 | --- | --- | --- | --- |
 | cert-manager| cpu: 10m<br />memory: 32Mi |  | Yes |
```

Comment:
gatekeeper comes with 3 pods by default, each with 100m CPU request and 256 Mi memory requests.

```
| gatekeeper | cpu: 300m<br />memory: 768Mi | | Yes |
```

(going to make these comments one by one in case other people are reviewing)